crates/fuel-core/src/graphql_api/indexation/balances.rs

```diff
@@ -0,0 +1,592 @@
+use fuel_core_storage::StorageAsMut;
```

Comment:
This file is almost the same as `crates/fuel-core/src/graphql_api/indexation.rs`(compared with the original file), except moving balance related errors to `error` module and small renaming.

Skipping this file from reviewing.

Reply:
Yes, even in the new PR, the logic around the "balances indexation" has not been changed. There are minor changes like variable renames.